pkg/common/configs/configvalidator_test.go

```diff
@@ -2012,3 +2012,227 @@ func TestCheckQueuesStructure(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckQueues(t *testing.T) { //nolint:funlen
+	testCases := []struct {
+		name             string
+		queue            *QueueConfig
+		level            int
+		errorExpected    bool
```

Comment:
It seems `validateFunc` is not called if `errorExpected` is true. Maybe we can merge them to simplify code.

Reply:
Update the code to remove the errorExpected variable and simplify code. Many thanks for the suggestion.